content/articles/txt-record.markdown

```diff
@@ -1,21 +1,83 @@
+{:toc}  
+
+---
+
+## What's a TXT Record?
 
 A TXT record is a resource record used to provide the ability to associate text with a zone. This record allows domain administrators to insert any text content into DNS records. These records are used for various purposes. One example is ownership validation: To prove you own the domain, a provider may require you to add a TXT record with a particular value to your domain.
 
 Common uses for TXT records:
 
-| Type | Description |
-|------|-------------|
-| [`DKIM` records](/articles/dkim-record) | This record stores important information used in the validation of email in transit. |
-| DMARC records | Domain-based Message Authentication Reporting and Conformance records mitigate phishing and spoofing email attacks. |
-| [`SPF` record](/articles/spf-record/) | This record is used to indicate to mail exchanges which hosts are authorized to send mail for a domain. TXT records should be used instead of the special SPF record type. |
-| Site Verification Records | This record proves ownership of a domain and can be used to associate services such as Microsoft 365 and G-Suite to a specific domain. |
+| Type                                    | Description                                                                                                                                                                |
+|-----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`DKIM` records](/articles/dkim-record) | This record stores important information used in the validation of email in transit.                                                                                       |
+| DMARC records                           | Domain-based Message Authentication Reporting and Conformance records mitigate phishing and spoofing email attacks.                                                        |
+| [`SPF` record](/articles/spf-record/)   | This record is used to indicate to mail exchanges which hosts are authorized to send mail for a domain. TXT records should be used instead of the special SPF record type. |
+| Site Verification Records               | This record proves ownership of a domain and can be used to associate services such as Microsoft 365 and G-Suite to a specific domain.                                     |
+
+## Validation
+
+We consider two different validation scenarios:
+
+### When you provide a deserialized TXT record content
+
+We will consider this validation scenario when the TXT record content you provide is not wrapped in double quotes. This is the most common scenario.
+
+In this scenario, there are few limitations on what you can do beyond a hard limit of 1000 characters for the serialized version of your content. See the next section to understand what we mean about the serialized version of the TXT record content.
```

Comment:
Maybe it's worth explaining that in this scenario we will handle the normalization of the the content.

Reply:
Good idea!